### PR TITLE
Fix stuff after 2038

### PR DESCRIPTION
**Pull request by LefterisJP**
Fixing some problems I found with post-merge review of #2038 

[ci integration]

### CODE REVIEW

File: raiden/transfer/state.py [hunk 1]
```diff
@@ -186,7 +186,7 @@ def __ne__(self, other):
 
 
 class TokenNetworkState(State):
-    """ Corresponds to a channel manager smart contract. """
+    """ Corresponds to a token newtwork smart contract. """
```
Comment (palango) [inline raiden/transfer/state.py]: s/newtwork/network

Comment (LefterisJP) [reply raiden/transfer/state.py]: Token NEWTORK is what I intended to write obviously ....

Comment (palango) [reply raiden/transfer/state.py]: Sorry, my bad 🙊

**Comment by LefterisJP** [comment]
All tests passed apart from some weird [matrix test failures](https://travis-ci.org/raiden-network/raiden/jobs/413610552) which don't seem related to this PR.

 I will fix the typo and merge.

**Pull request merged**